prelude/size.md

```diff
@@ -28,5 +26,42 @@ impl Size {
 
     pub fn bytes(self) -> BigInt { self.raw }
     pub fn bits(self) -> BigInt { self.raw * 8 }
+    pub fn is_zero(&self) -> bool { self.raw == 0 }
+}
+
+// We implement a few operators for size.
```

Comment:
Soo, I decided to implement a few operators for `Size`.
It doesn't feel great having that in the spec really.

But: It feels even worse writing `Size::from_bytes(a.bytes() + b.bytes())` instead of `a+b` over and over again.
and if `specr` is a crate, it cannot secretly implement `Size`s operators.

Reply:
Also, I can extract this into another PR if you want. Doesn't really qualify as "fixes" ^^

Reply:
Yeah this will definitely not translate well to Coq...

... I think you convinced me that Size and Align should be defined in specr, not in MiniRust.